src/Auth0.ManagementApi/Clients/UsersClient.cs

```diff
@@ -286,5 +286,47 @@ public Task<User> UpdateAsync(string id, UserUpdateRequest request)
                 {"id", id}
             });
         }
+
+        /// <summary>
+        /// Get the permissions assigned to the user.
+        /// </summary>
+        /// <param name="id">The id of the user to obtain the permissions for.</param>
```

Comment:
"id" => "ID"

Reply:
There is a mix of both throughout the codebase.  Probably deserves a separate clean-up PR.